knowledge/responses/what_about_protein.md

```diff
@@ -1,17 +1,18 @@
+To counter the claim that there is an inefficient amount of protein in a vegan diet, it is pertinent to include areas where a non-vegan diet can actually cause a reduction in protein intake. One such area, which reduces the liver's protein production, is inflammation. Inflammation can lower protein levels because of albumin, a negative acute phase protein. When inflammation occurs, the liver, which produces albumin, must shift from that albumin production to the production of C reactive proteins, which are positive acute phase proteins, to fight the inflammation.[4] Not only do vegans have lower C reactive protein risk, but when non-vegans are put on a vegan diet, their C reactive protein levels can lower by one-third [5]. This results in higher levels of available blood protein for someone on a whole plant-based diet because a lot of animal products and processed foods increases risk of inflammation [4], which forces the liver to shift from general protein production to inflammation of specific proteins. Although this is just an example, it showcases how a diet centered around animals, which is often synonymous with protein itself, can actually cause issues with a reduction of total protein levels, not increase it [4].
 
-The overall take from all this is that **animals are not a required source of protein.** A plant based diet can have enough protein for any requirement someone may have, and a whole plant food diet can even be more beneficial to protein levels compared to a diet focused on meat or processed foods. Although some research may be needed for individuals with high protein needs, animals are ultimately unneeded, and **the average person will not need to supplement protein, or have protein deficiencies, if they base their diet on whole plant sources.**
+The overall take from all this is that animals are not a required source of protein. A plant-based diet can have sufficient protein, and a whole food plant-based diet can be even more beneficial for protein levels compared to a diet focused on meat or processed foods [6]. Although some research may be needed for individuals with high protein needs, animals are ultimately unneeded, and the average person will not need to supplement protein, or have protein deficiencies, if they base their diet on whole plant sources.
 
```

Comment:
We're missing a response to another very common protein myth, which is the _quality_ of vegan proteins do not measure up to animal proteins. And that it is worse, you need to eat more to have the same effect or something of that nature. Here's a source that shows that there are no significant differences in consuming plant proteins and animal proteins for muscle mass and strength: Mangano, K. M. et al. ‘Dietary protein is associated with musculoskeletal health independently of dietary pattern: the Framingham Third Generation Study’, American Journal of Clinical Nutrition 105, 3: 714–22 (2017). 

Got this off Earthling Ed's new book.

Reply:
Addressed.